mobile/config.xml

```diff
@@ -27,8 +27,8 @@
         <preference name="StatusBarOverlaysWebView" value="false" />
         <allow-intent href="itms:*" />
         <allow-intent href="itms-apps:*" />
-        <config-file target="*-Info.plist" parent="ITSAppUsesNonExemptEncryption" mode="add">
-            <false/>
+        <config-file mode="add" parent="ITSAppUsesNonExemptEncryption" target="*-Info.plist">
```

Comment:
no need for this change. Could you use `git add -p` to select what changes you include in the commit?